README.md

```diff
@@ -106,7 +106,14 @@ If the PEP Proxy is not started as a service, it can be started executing the fo
 bin/pep-proxy.js
 ```
 
-Once the PEP Proxy is working, it can be used to enforce both authentication and authorization over the protected component (e.g. Orion Context Broker). In order to enforce both actions, the PEP Proxy has to be connected to an [Identity Manager](https://github.com/ging/fi-ware-idm) server and an [Access Manager](https://github.com/telefonicaid/fiware-keypass) server. Next sections will show some examples of both processes
+Once the PEP Proxy is working, it can be used to enforce both authentication and authorization over the protected component (e.g. Orion Context Broker). In order to enforce both actions, the PEP Proxy has to be connected to an [Identity Manager](https://github.com/ging/fi-ware-idm) server and an [Access Manager](https://github.com/telefonicaid/fiware-keypass) server. Next sections will show some examples of both processes.
+
+Note that, in order for a request to be processed, it's mandatory that it contains the all of following headers:
+* **x-auth-token**: should contain a valid user token, generated by the IDM.
+* **fiware-service**: should contain the name of a service created in the IDM and the user must have access to it.
+* **fiware-servicepath**: should contain a route to a subservice, begining with a slash '/' symbol.
```

Comment:
It's a possibility, but, considering the PEP will be enforcing this header for all the components of the IoT Platform, that decision should be taken for all the platform, rather than for a single component. Should be better discussed offline.

Agree with the NTC.